docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,51 @@
+# Release 460 (2 Oct 2024)
+
+## Atop connector
+
+* {{breaking}} Remove the Atop connector. ({issue}`23550`)
+
+## ClickHouse connector
+
+* Improve performance of listing columns. ({issue}`23429`)
+* Improve performance for queries comparing `varchar` columns. ({issue}`23558`) 
+* Improve performance for queries using `varchar` columns for `IN` comparisons. ({issue}`23581`)
+* Improve performance for queries with complex expression for `LIKE` clauses. ({issue}`23591`)
+
+## Delta Lake connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hive connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hudi connector
+
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Iceberg connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Ensure table columns are cached in Glue even when table comment is too long.
+  ({issue}`23483`)
+* Fix queries stuck in the planning phase for a long time when there are large
```

Comment:
@martint @raunaqmorarka ?

Reply:
Maybe:
Reduce planning time for queries on tables with a large number of structural type fields

Reply:
Done